### PR TITLE
Refactor/css styling and readme

### PR DESCRIPTION
**Pull request by Denzel-Turambi**
What I did:
- refactor styling rule for budget-input, submit button, and nav bar
- Update preview gif to be the most up-to-date version of our application

Did this break anything?
- [x] No

Type of change
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [x]  Super small fix (Corrected a typo, removed a comment, etc.)

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ In order to run this application:
 7. Copy and paste that local host link into web browser.
 
 ### Preview of App:
-![Preview](https://user-images.githubusercontent.com/122255250/240078072-ccc14e54-2365-420b-a4ee-396977f2f508.gif)
+![Preview](https://user-images.githubusercontent.com/122255250/243220061-ddd7a47a-c428-4143-809d-8ffae090de3a.gif)
```
Comment (andreasorensen) [inline README.md:17]: Nice!

File: src/styles.css [hunk 2]
```diff
@@ -26,6 +26,7 @@ nav {
   justify-content: center;
   align-items: center;
   justify-content: space-around;
+  box-shadow: 0 4px 20px -2px #A2AEBB;
```
Comment (andreasorensen) [inline src/styles.css:29]: Ooooh. Looks interesting!

**Review by andreasorensen — APPROVED** [review]
We're almost there! Great job Denzel. Thanks for making those changes.

**Review by hvohr — APPROVED** [review]
thank you for doing this, this is super clean code

**Review by fernandorobles97 — APPROVED** [review]

**Pull request merged**